dss/storage/validator.py

```diff
@@ -0,0 +1,78 @@
+"""Utilities in this file are used to removed extra fields from index data before adding to the index."""
+import re
+from jsonschema.validators import create, RefResolver
+from jsonschema import _utils, _validators
+
+import json
+from urllib.request import urlopen
+from urllib.error import URLError
+import ssl
+from typing import List
+import logging
+
+
+def read_json_url(link):
+    gcontext = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
+    f = urlopen(link, context=gcontext)
+    return json.load(f)
+
+
+def extract_json_fields(json_data: dict, path: List[str], fields: List[str]):
```

Comment:
"extract" doesn't necessarily imply removal.

Reply:
changed to `remove_json_fields`.